src/test/java/com/gizasystems/automationexercise/tests/login/LoginWithCorrectEmailAndPasswordTest.java

```diff
@@ -0,0 +1,65 @@
+package com.gizasystems.automationexercise.tests.login;
+
+import com.gizasystems.automationexercise.apis.Apis;
+import com.gizasystems.automationexercise.apis.ApisAccountManagement;
+import com.gizasystems.automationexercise.pages.*;
+import com.shaft.driver.SHAFT;
+import io.qameta.allure.Description;
+import io.qameta.allure.Epic;
+import io.qameta.allure.Feature;
+import io.qameta.allure.Story;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class LoginWithCorrectEmailAndPasswordTest {
+    private SHAFT.GUI.WebDriver driver;
+    private SHAFT.API api;
+    private SHAFT.TestData.JSON registerTestData;
+    private SHAFT.TestData.JSON loginTestData;
+    private String timeStamp = String.valueOf(System.currentTimeMillis());
+
+    @Test(description = "Login With Correct Email and Password Test - Register and Delete user with APIs ")
+    @Description("Given that I am a registered user, When I enter correct email, And Correct Password , And I Clicked on Login button And I clicked on delete user, Then I should be Logged in successfully to the system, And user be deleted from the system")
+    public void LoginWithCorrectEmailAndPassword() {
+        new ApisAccountManagement(api)
+                .createRegisterUserAccount(registerTestData.getTestData("UserName"), registerTestData.getTestData("UserMail.ApiTimeStamp") + timeStamp + "@gizasystems.com", registerTestData.getTestData("UserPassword"), registerTestData.getTestData("UserFirstName"), registerTestData.getTestData("UserLastName"))
+                .validateUserCreatedRegistered();
+        new NavigationBar(driver)
+                .clickOnSignupLoginLink();
+        new SignupLoginPage(driver)
+                .validateOnLoginVisibility(loginTestData.getTestData("Messages.Login"))
+                .registeredUserLogin(registerTestData.getTestData("UserMail.ApiTimeStamp")+ timeStamp + "@gizasystems.com",registerTestData.getTestData("UserPassword"));
+        new NavigationBar(driver)
+                .validateTheLoggedInUser(registerTestData.getTestData("UserName"));
+        new ApisAccountManagement(api)
+                .deleteUserAccount(registerTestData.getTestData("UserMail.ApiTimeStamp")+ timeStamp + "@gizasystems.com",registerTestData.getTestData("UserPassword"))
+                .validateDeleteUser()
+                .validateUserNotFound(registerTestData.getTestData("UserMail.ApiTimeStamp")+ timeStamp + "@gizasystems.com");
+
+    }
+
+
+    //////////////////// Configurations \\\\\\\\\\\\\\\\\\\\
+    @BeforeClass
+    public void beforeClass() {
+        registerTestData = new SHAFT.TestData.JSON("src/test/resources/testDataFiles/RegisterUser.json");
```

Comment:
We should only have 1 to 1 relationship between the test data json file and test class.
Please add the registration data in the corresponding test data json file with unique names to the test case.